### PR TITLE
Homework4: Olga Chubryk

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/ochubryk/hometask4/cities.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Дан список стран и городов каждой страны.
```
Comment (belovmd) [inline src/ochubryk/hometask4/cities.py:1]: would it work if i'll enter Brest in Belarus and France?

File: src/ochubryk/hometask4/unique_numbers.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Даны два списка чисел. Посчитайте, сколько
+# чисел входит только в один из этих списков
+
+a = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+b = [11, 12, 13, 3, 4, 5, 15, 16, 7]
+
+# print(len(set.difference(set(a), set(b))))
```
Comment (belovmd) [inline src/ochubryk/hometask4/unique_numbers.py]: redundant comment

File: src/ochubryk/hometask4/Xs_and_Os_referee.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+# A game's result is presented as a list of strings,
+# where "X" and "O" are players' marks and "." is the empty cell.
+
+
+from typing import List
+
+
+def checkio(game_result: List[str]) -> str:
+    for i in range(3):
+        if game_result[i][0] == game_result[i][1] == game_result[i][2]\
```
Comment (belovmd) [inline src/ochubryk/hometask4/Xs_and_Os_referee.py]: in general these checks looks pretty bulky

**Review by belovmd — COMMENTED** [review]
I really like your naming. please take a look at these comments:

File: src/ochubryk/hometask4/cities.py (hunk 1 above)
Comment (ghost) [reply src/ochubryk/hometask4/cities.py:1]: It's all OK. This is the execution of the program below

image

File: src/ochubryk/hometask4/unique_numbers.py (hunk 2 above)
Comment (ghost) [reply src/ochubryk/hometask4/unique_numbers.py]: Fixed

File: src/ochubryk/hometask4/Xs_and_Os_referee.py (hunk 3 above)
Comment (ghost) [reply src/ochubryk/hometask4/Xs_and_Os_referee.py]: Added new better solution
Fixed

File: src/ochubryk/hometask4/cities.py (hunk 1 above)
Comment (belovmd) [reply src/ochubryk/hometask4/cities.py:1]: 👍

File: src/ochubryk/hometask4/Xs_and_Os_referee.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+
+from typing import List
+
+
+def checkio(game_result: List[str]):
+    board_size = len(game_result)
+    rows = game_result
+    columns = list(map(''.join, zip(*rows)))
+    diagonals = list(map(''.join, zip(*[(row[i], row[board_size - 1 - i])
```
Comment (belovmd) [inline src/ochubryk/hometask4/Xs_and_Os_referee.py:25]: list map zip enumerate in one command ^)))))

**Review by belovmd — COMMENTED** [review]
9,5

**Pull request closed without merging**